README.md

```diff
@@ -94,7 +94,7 @@ Check the current [open issues](https://github.com/zowe/vscode-extension-for-zow
 
 To report a bug or request a specific feature, please open a GitHub issue using the [appropriate template](https://github.com/zowe/vscode-extension-for-zowe/issues/new/choose). Feature requests will be added to our backlog after it receives 10 upvotes from the community.
 
-Also, you can check our [Zenhub Communities boards](https://github.com/zowe/vscode-extension-for-zowe#workspaces/zowe-cli-explorers-5d77ca38fb288f0001ceae92/board?repos=150100207) for a more convenient view of issues and access to other boards of Zowe-related projects.
+Also, you can check our [GitHub Projects board](https://github.com/orgs/zowe/projects/15) for a more convenient view of issues that is categorized by priority and shows work in progress.
```

Comment:
there's a little knot here w/ the tenses. how about:
"... for a more convenient view of issues, which are categorized by priority and organized by work-in-progress status."

or:

"Also, you can view issues that are work in progress, and their assigned priority, in our GitHub Projects board."